examples/samplers/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -4,15 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>Typo in DEMetropolis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>revised</p><p> </p>